packages/netdog/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "netdog"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
```

Comment:
👍

Reply:
> To avoid also triggering rebuilds of os when netdog changes, you'll need to relocate sources/api/netdog to sources/netdog.

Why is this, do we have a watch on all of `sources/api`? Incidentally, I don't think I've never seen a variant build in which os is not rebuilt.

Reply:
This was done